sites/hurl.dev/_docs/request.md

```diff
@@ -7,16 +7,56 @@ section: File Format
 
 ## Definition {#definition}
 
-Describes an HTTP request, with mandatory [method] and [url], followed by optional [headers],
-[query parameters], [form parameters], [multipart form datas], [cookies], 
-[basic authentification] and [body].
+Describes an HTTP request: a mandatory [method] and [url], followed by optional [headers].
+
+Then, [query parameters], [form parameters], [multipart form datas], [cookies] and
+[basic authentification] can be used to configure the HTTP request.
+
+Finally, an optional [body] can be used to configure the HTTP request body.
 
 ## Example {#example}
 
 ```hurl
 GET https://example.net/api/dogs?id=4567
 User-Agent: My User Agent
 Content-Type: application/json
+[BasicAuth]
+alice: secret
+```
+
+[Headers], if present, follow directly [method] and [url]. This allows Hurl format to 'looks like' the real HTTP format.
```

Comment:
"...follow directly after the [method] and [url]"

"...to 'look like' the real HTTP format"